pkg/endpointmanager/manager.go

```diff
@@ -156,7 +156,7 @@ func (mgr *EndpointManager) InitMetrics() {
 		// It must be thread-safe.
 		metrics.EndpointCount = prometheus.NewGaugeFunc(prometheus.GaugeOpts{
 			Namespace: metrics.Namespace,
-			Name:      "endpoint_count",
+			Name:      "endpoint",
```

Comment:
One question, I don't see this metric `endpoint_count`/`endpoint` in the default metrics list https://github.com/cilium/cilium/pull/12826/files#diff-7cbe8e056d62a2de30c7066e359bd9c9R461 or grafana and prometheus example config. Is there a reason for this? cc @aanm

Reply:
no, it's likely forgotten